app/assets/javascripts/short-form/ShortFormApplicationService.js.coffee

```diff
@@ -588,7 +585,7 @@ ShortFormApplicationService = (
       # special case for contact form
       if stateName.match(/contact/)
         applicant = Service.applicant
-        addressValid = applicant.preferenceAddressMatch != null
+        addressValid = !_.isNil(applicant.preferenceAddressMatch)
```

Comment:
@creativecoder need to think through the logic here and make sure that intention.meaning is being preserved

Reply:
@callaghanc I think this makes sense--we want to know if the address has already been geocoded with a valid value, meaning `preferenceAddressMatch` should have a value other than `null` or `undefined`.